py-rse/install.Rmd

```diff
@@ -0,0 +1,3 @@
+# Setting Up {#py-rse-install}
+
+FIXME: write installation instructions.
```

Comment:
Sorry, I'm not sure what this means. To install the book and dependencies? For another chapter?